carpool/views.py

```diff
@@ -47,12 +48,15 @@ def new(request):
             })
             to_email = form.cleaned_data.get('email')
             send_mail(mail_subject, message, 'Hackweek@example.com', [to_email], fail_silently=False)
-            done=True
+            HttpResponseRedirect("{% url 'polls.signupcomp' %}")
             # email = EmailMessage(mail_subject, message, to=[to_email])
             # email.send()
     else:
         form = SignUpForm()
-    return render(request, 'signup.html', {'form': form, 'error': error, 'done': done, })
```

Comment:
It is better to have a combined `render` rather than using the same code three times, just remove the `done` variable.